### PR TITLE
Rename Element to AutomationElement

### PR DESCRIPTION
**Pull request by maxinfet**
This change is to make it easier to convert code from System.Windows.Automation to FlaUI.

### CODE REVIEW

**Comment by maxinfet** [comment]
Closed pull request https://github.com/Roemer/FlaUI/pull/8 and https://github.com/Roemer/FlaUI/pull/10 because I messed them up

File: src/FlaUI.Core.UITests/Elements/TabTests.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public void TabSelectTest()
             RestartApp();
             var mainWindow = App.GetMainWindow(Uia3Automation);
             var tab = mainWindow.FindFirst(TreeScope.Descendants, mainWindow.ConditionFactory.ByControlType(ControlType.Tab)).AsTab();
-            Assert.That(tab.TabItems, Has.Length.EqualTo(2));
+            Assert.That(tab.TabItemsAutomation, Has.Length.EqualTo(2));
```
Comment (Roemer) [inline src/FlaUI.Core.UITests/Elements/TabTests.cs]: I think this should still be TabItems?

File: src/FlaUI.Core/Elements/Tab.cs [hunk 2]
```diff
@@ -5,49 +5,49 @@
...
         public Tab(AutomationObjectBase automationObject) : base(automationObject)
         {
         }
 
         /// <summary>
-        /// The currently selected <see cref="TabItem"/>
+        /// The currently selected <see cref="TabItemAutomation"/>
         /// </summary>
-        public TabItem SelectedTabItem
+        public TabItemAutomation SelectedTabItemAutomation
```
Comment (Roemer) [inline src/FlaUI.Core/Elements/Tab.cs]: Should be SelectedTabItem

File: src/FlaUI.Core/Elements/Tab.cs [hunk 3]
```diff
@@ -5,49 +5,49 @@
...
+        /// The index of the currently selected <see cref="TabItemAutomation"/>
         /// </summary>
         public int SelectedTabItemIndex => GetIndexOfSelectedTabItem();
 
         /// <summary>
-        /// All <see cref="TabItem"/> objects from this <see cref="Tab"/>
+        /// All <see cref="TabItemAutomation"/> objects from this <see cref="Tab"/>
         /// </summary>
-        public TabItem[] TabItems => GetTabItems();
+        public TabItemAutomation[] TabItemsAutomation => GetTabItems();
```
Comment (Roemer) [inline src/FlaUI.Core/Elements/Tab.cs]: Should be TabItem / TabItems

File: src/FlaUInspect/Properties/Resources.resx [hunk 4]
```diff
@@ -60,47 +60,47 @@
             : and then encoded with base64 encoding.
     -->
   <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
-    <xsd:element name="root" msdata:IsDataSet="true">
+    <xsd:automationElement name="root" msdata:IsDataSet="true">
```
Comment (Roemer) [inline src/FlaUInspect/Properties/Resources.resx]: This seems wrong in the whole file. I think here it should stay element.

**Comment by maxinfet** [comment]
@Roemer Resolved the issues you reported.

File: src/FlaUInspect/Properties/Resources.resx (hunk 4 above)
Comment (maxinfet) [inline src/FlaUInspect/Properties/Resources.resx]: Switched this back to element. Resharper named this and I didn't notice.

File: src/FlaUI.Core/Elements/Tab.cs (hunk 3 above)
Comment (maxinfet) [inline src/FlaUI.Core/Elements/Tab.cs]: Switched all TabItemAutomation and TabItemsAutomation back to TabItem and TabItems respectively. Resharper renamed this and I didn't notice.

File: src/FlaUI.Core/Elements/Tab.cs (hunk 2 above)
Comment (maxinfet) [inline src/FlaUI.Core/Elements/Tab.cs]: Switched all TabItemAutomation and TabItemsAutomation back to TabItem and TabItems respectively. Resharper renamed this and I didn't notice.

**Comment by Roemer** [comment]
Merged this one manually as well because of the merge conflicts.

**Pull request closed without merging**